<?xml version="1.0"?>
<launch>

    <!-- LAUNCH INTERFACE -->
    <arg name="hardware_interface" default="PositionJointInterface"/>

    <!-- LAUNCH IMPLEMENTATION -->
    <rosparam command="load" file="$(find iiwa_hw_fri)/config/joint_names.yaml" />
    <!-- addresses /-->
    <param name="interface" value="$(arg hardware_interface)"/>

    <!-- the real hardware interface /-->
    <node name="iiwa_hw" pkg="iiwa_hw_fri" type="iiwa_hw_fri-bin" respawn="false" output="screen"/>

</launch>
